packages/dev-frontend/src/components/ReactModal.tsx

```diff
@@ -11,7 +11,8 @@ const modalStyle = {
     transform: "translate(-50%, -50%)",
     maxWidth: "620px",
     width: "580px",
-    maxHeight: "94vh"
+    maxHeight: "94vh",
+    overflow: "visible"
```

Comment:
Tooltips get hidden (in Chrome) when displayed inside react-modal, due to its default overflow:auto style.